src/content/guides/creating-token-collections.mdx

```diff
@@ -0,0 +1,48 @@
+---
+title: Creating Token Collections
+description: Create token collections, define redemption conditions, and mint tokens for the collections.
+nav:
+  path: Reference/Digital Assets
+  order: 3
+---
+
+## What is a Digital Token?
+
+Digital tokens are assets that can be exchanged for specific products or deals. Think a "Free coffee" coupon or a "50% off" voucher.
+Unlike assets such as gift cards or currency, tokens can only be spent in their entirety, all at once.
+
+### Token Collection
+
+A collection is a grouping of tokens that share the same attributes and redemption conditions, e.g.
+
+- Name
+- Icon
+- Expiry period
+- Maximum redemption value
+
+### Redemption Conditions
+
+A redemption condition allows a merchant to accept tokens from a token collection.
```

Comment:
@seliwe Suggest capitalising `Merchant` and linking it to the resource in the API docs

Reply:
Nice will do